lib/bom.js

```diff
@@ -874,7 +901,7 @@ declare class BiquadFilterNode extends AudioNode {
   Q: AudioParam;
   gain: AudioParam;
   type: 'lowpass'|'highpass'|'bandpass'|'lowshelf'|'highshelf'|'peaking'|'notch'|'allpass';
-  getFrequencyResponse(frequencyHz: Float32Array, magResponse: Float32Array, phaseResponse: Float32Array): BiquadFilterNode;
```

Comment:
`getFrequencyResponse` should return `void`
https://webaudio.github.io/web-audio-api/#dom-biquadfilternode-getfrequencyresponse